product_docs/docs/transporter/ba_preview/getting_started/installing/index.mdx

```diff
@@ -6,32 +6,28 @@ navigation:
   - linux_x86_64
```

Comment:
We have added docs for Linux x86, but nothing for ARM64, nor ppc64le, which I think is intended like that. Just commenting here in case it is not intended. We might consider adding a note in this index.mdx page saying that at this time, we only support x86 distros, but it is optional.